.github/workflows/build_test_deploy.yml

```diff
@@ -20,10 +20,19 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
+        with:
+          fetch-depth: 25
       - name: Install moreutils
         run: sudo apt install moreutils
       - name: Check examples
         run: ./scripts/check-examples.sh
+      - name: Install Vercel CLI
+        run: npm i -g vercel@28.16.15
+      - name: Deploy examples
+        run: ./scripts/deploy-examples.sh
```

Comment:
Do we want to add an inverse `if` condition here for not `isRelease`?

Reply:
Hmm good point. How does this look: [aa8146a](https://github.com/vercel/next.js/pull/47130/commits/aa8146ab7bbdf383f27f7e384e16b9cd741b721a)